### DISCUSSION TITLE
Regexp for IPv6 addresses

### DISCUSSION THREAD

**Discussion by cayblood** [ipaddress-gem/ipaddress#134] [Ideas]
I noticed you have a REGEXP constant for ipv4 addresses. Do you have an equivalent for ipv6 addresses? Seems like that would be good to have too.

**Comment by sandstrom** [comment]
There is one here:
https://github.com/ipaddress-gem/ipaddress/blob/v0.8.3/lib/ipaddress.rb#L167
